Makefile

```diff
@@ -13,46 +13,46 @@ endif
 ifeq ($(GOARCH), aarch64)
 	GOARCH=arm64
 endif
-PHP_BINARY_PATH := legacy/archives/php_$(GOOS)_$(GOARCH)
+PHP_BINARY_PATH := ./internal/legacy/archives/php_$(GOOS)_$(GOARCH)
 
-legacy/archives/platform.phar:
-	wget https://github.com/platformsh/legacy-cli/releases/download/v$(PSH_VERSION)/platform.phar -O legacy/archives/platform.phar
+./internal/legacy/archives/platform.phar:
+	wget https://github.com/platformsh/legacy-cli/releases/download/v$(PSH_VERSION)/platform.phar -O ./internal/legacy/archives/platform.phar
 
-legacy/archives/php_windows_amd64: legacy/archives/php_windows.zip legacy/archives/cacert.pem
+./internal/legacy/archives/php_windows_amd64: ./internal/legacy/archives/php_windows.zip ./internal/legacy/archives/cacert.pem
 
-legacy/archives/php_darwin_$(GOARCH):
+./internal/legacy/archives/php_darwin_$(GOARCH):
 	bash build-php-brew.sh $(PHP_VERSION) $(GOOS)
 	mv -f $(GOOS)/php-$(PHP_VERSION)/sapi/cli/php $(PHP_BINARY_PATH)
 	rm -rf $(GOOS)
 
-legacy/archives/php_linux_$(GOARCH):
+./internal/legacy/archives/php_linux_$(GOARCH):
 	cp ext/extensions.txt ext/static-php-cli/docker
 	cd ext/static-php-cli/docker ;\
 	sed -i 's/alpine:latest/alpine:3.16/g' Dockerfile;\
 	docker build --platform=linux/$(GOARCH) -t static-php . --build-arg USE_BACKUP_ADDRESS=yes --progress=plain
-	mkdir -p legacy/archives
-	docker run --rm --platform=linux/$(GOARCH) -v ${PWD}/legacy/archives:/dist -e USE_BACKUP_ADDRESS=yes static-php build-php no-mirror $(PHP_VERSION) all /dist
-	mv -f legacy/archives/php legacy/archives/php_linux_$(GOARCH)
-	rm -f legacy/archives/micro.sfx
+	mkdir -p ./internal/legacy/archives
+	docker run --rm --platform=linux/$(GOARCH) -v ${PWD}/./internal/legacy/archives:/dist -e USE_BACKUP_ADDRESS=yes static-php build-php no-mirror $(PHP_VERSION) all /dist
```

Comment:
is it necessary to start paths with `./` everywhere in this file? it looks redundant. especially in the line above ⬆️

Reply:
Not really, personal preference. I tuned this down please take another look.